downloader/src/main/kotlin/Downloader.kt

```diff
@@ -283,7 +283,7 @@ class Downloader(private val config: DownloaderConfiguration) {
             path = pkg.vcsProcessed.path
         )
 
-        return RepositoryProvenance(vcsInfo, pkg.vcsProcessed.takeIf { it != vcsInfo })
+        return RepositoryProvenance(vcsInfo, pkg.vcsProcessed)
```

Comment:
What would help me to understand better:

- Please explain why "`pkg.vcsProcessed.resolvedRevision` is always blank" is actually correct (because `resolvedRevision` is only populated by the downloader).
- Please explain why "`vcsInfo.resolvedRevision` is never blank" is actually correct.
- In exchange drop the last paragraph.